plugin/ochttp/example_test.go

```diff
@@ -45,22 +50,51 @@ func ExampleTransport() {
 	client := &http.Client{
 		Transport: &ochttp.Transport{},
 	}
-	_ = client // use client to perform requests
+
+	// Use client to perform requests.
+	_ = client
 }
 
 var usersHandler http.Handler
 
 func ExampleHandler() {
-	// Enables OpenCensus for the default serve mux.
-	// By default, B3 propagation is used.
-	http.Handle("/users", usersHandler)
+	// import "go.opencensus.io/plugin/ochttp"
+
+	ochttp.Handle(http.DefaultServeMux, "/users", usersHandler)
+
+	// If no handler is specified, the default mux is used.
 	log.Fatal(http.ListenAndServe("localhost:8080", &ochttp.Handler{}))
 }
 
 func ExampleHandler_mux() {
+	// import "go.opencensus.io/plugin/ochttp"
+
+	mux := http.NewServeMux()
+	ochttp.Handle(mux, "/users", usersHandler)
+	log.Fatal(http.ListenAndServe("localhost:8080", &ochttp.Handler{
+		Handler:     mux,
+		Propagation: &b3.HTTPFormat{},
+	}))
+}
+
+var keyUserId, _ = tag.NewKey("user_id")
+
+func ExampleTaggedHandlerFunc() {
+	// import "go.opencensus.io/plugin/ochttp"
+
+	// var keyUserId, _ = tag.NewKey("user_id")
+
 	mux := http.NewServeMux()
-	mux.Handle("/users", usersHandler)
+	ochttp.Handle(mux, "/users", ochttp.TaggedHandlerFunc(func(w http.ResponseWriter, r *http.Request) []tag.Mutator {
```

Comment:
@rakyll not quite ready for review but wanted to get your feedback on this. We discussed before a way to allow tags to be added to the record call at the end of the request. This way avoids exposing any mutable context keys (although it uses a mutable context key underneath to implement it).